custom_validators/pubkeys.json

```diff
@@ -0,0 +1 @@
+["0xad3787b26dd7a54442ecb28ce590de27fcb19f20bef7e5aadee7b1a8572b8f30eff9717ff5e3d92b313891f63faf3076","0x910c7ccf7586a85c9d80fa785c2aea372754c8c2a6e75e4e366b30cff34e52c39366b832a37337d72859d60aaf028a4c","0xae5c8216d20b1dcdab0461af9b115ecf783f790e23ca95f86f231095a5079edd4465429c696e7f8688d364c071f993fc","0x81da961072378fe7839b1d4c4d508d64fd57f9242ee64dc13e1f3899a0a55aec2d5188681da351c41ba833e9b1050057","0xb219fa26c3c55967a827d4e38b78bf559d551e085814ea2dd952eadb40d08f37c1e96f3435db65acd105e031045e1ae3","0xb0661fb09e80d346083cfbdbdaf8235efe15b5501abfb2d997cd4b2f23bcf67d0c869b60a76b16fa1899f2a5f3f72a2d","0xa31a4b5e1ff869bec30e4d913ff254370224bbfc4f309b33c80e5a899619c1856877c699490e95c0bdbdd3e98d5b5808","0x94ce1966bc90eb29e5029c660b7f649711582533f6cd067eabb3b5d29e4f93b17868af8832a0f614223e20685d967209","0xaae05d47a22220006b446c412403f9aebfe0aca528e8e1e038ca41f3e17c4656fa88df7f50ad385e7dc946322d862dd4","0xa3a04f0fc7b464d7c4c3ad55a002a2a8fc13014047972b9313c12c675b24ae56f9067d9ab0a41919a093c63302ee4115","0x80003b5919fee8fcec61cdc96aeab4b3bd0e6fa8632ec5609a92bd51dd5be9c68b79b7437de0c2ffe9da91e9d4c8d85c"]
```

Comment:
I think its better to have a raw text file with 1 line 1 key. easier to parse, less error prone and easier to read.